modules/rhcos-load-firmware-blobs.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/machine-configuration-tasks.adoc
+
+[id="rhcos-load-firmware-blobs_{context}"]
+
+= Loading custom firmware blobs in the machine config manifest
+
+Because the default location for firmware blobs in `/usr/lib` is read-only, you can locate a custom firmware blob by updating the search path. This enables you to load local firmware blobs in the machine config manifest when they are not managed by {op-system}.
+
+.Procedure
+
+. Create a Butane config file, `98-worker-firmware-blob.bu`, that updates the search path with the required settings. This example places the custom blob file outside of `/usr/lib`.
```

Comment:
It would be better to specify that this should be under a root-owned local-storage writable path, possibly with a real example like `/var/lib/firmware/`.

More importantly, the config example below is missing the crucial step where `firmware_class.path` is set through `kernelArguments`

Reply:
Thanks for clarifying. I've reworded throughout to use the suggested root-owned local storage path with the `/var/lib/firmware/` example. And I've added the karg to the config example. I'm not sure I have the wording down in the callouts, though, so would appreciate corrections/clarifications where necessary.